garage/logger/singleton_tabular.py

```diff
@@ -0,0 +1,91 @@
+"""Contains the singleton tabular class.
```

Comment:
It's not a singleton, it's just a class. I can make as many of them as I want.

Reply:
Fixed